### PR TITLE
Create a code_of_conduct.md file for issue #722

### LINKED ISSUE #722: Create a code_of_conduct.md file
**Is your feature request related to a problem? Please describe.**
As a developer ,
I need to update the repository with all proper links and documents.
1)Create a  new code_of_conduct.md file that has the same content as http://systers.io/code-of-conduct
2) And then update the code of conduct link in **Readme.md**>>Contributing section as well as in **Contributing.md**  .

### Acceptance Criteria

- [ ]  Update [Required]

### Definition of Done

- [ ]  All of the required items are completed.
- [ ]  Approval by 1 mentor.

### Estimation
30 min

### PR DESCRIPTION
**Pull request by sanchibansal340**
### Description
This pull request modifies the Code of Conduct according to the existing guidelines of http://systers.io/code-of-conduct and contact info has been changed according to https://github.com/anitab-org/stem-diverse-tv/blob/master/docs/code_of_conduct.md and has been added to /docs 

The links of Code of Conduct in README.md and CONTRIBUTING.md have been modified as well.

Fixes #722

### Type of Change:
Documentation

### How Has This Been Tested?

Tested on StackEdit.

### Checklist:

- [ ] I have performed a self-review of my own code or materials.

### CODE REVIEW

**Review by vj-codes — CHANGES_REQUESTED** [review]
@sanchibansal340 Congrats on your first contribution to the repository, great work !!!
Just a minor change, can you delete [this one](https://github.com/anitab-org/mentorship-backend/blob/develop/.github/code_of_conduct.md) please and then you are done.

**Comment by sanchibansal340** [comment]
Done. Thanks @vj-codes for helping me out in solving this issue.

**Review by vj-codes — DISMISSED** [review]
LGTM

File: docs/code_of_conduct.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
```
Comment (isabelcosta) [inline docs/code_of_conduct.md:2]: @sanchibansal340 could you make sure the Code of Conduct's content is consistent with this one https://github.com/anitab-org/documentation/blob/master/Contributing/CODE_OF_CONDUCT.md?

Comment (sanchibansal340) [reply docs/code_of_conduct.md:2]: @isabelcosta I will make the changes needed

**Review by isabelcosta — APPROVED** [review]
Great work @sanchibansal340 🎉

**Review by vj-codes — APPROVED** [review]
LGTM!

**Comment by isabelcosta** [comment]
Merged this because of it being a documentation issue.
Thank you @sanchibansal340 for your contribution!

**Comment by sanchibansal340** [comment]
Thank you :)

**Pull request merged**